frontend/src/app/main/data/viewer.cljs

```diff
@@ -77,55 +64,67 @@
     ptk/WatchEvent
     (watch [_ _ _]
       (rx/of (fetch-bundle params)
-             (fetch-comment-threads params)))))
+             (fetch-comment-threads params)))
+
+    ptk/EffectEvent
+    (effect [_ _ _]
+      (let [name (str "viewer-" file-id)]
+        (unchecked-set ug/global "name" name)))))
```

Comment:
What is this for?

Reply:
It allows redirect the user to the tab with a specific name, I mean, when you click edit, and you already have open a tab with workspace with that file, instead of opening a new tab, it will focus the already opened tab. It also works in a the other direction: workspace -> viewer.

Reply:
Interesting! This really deserves a comment :)

Reply:
I understand you set the window name here, and then you navigate to a target with this name, true?

If so, please explain it in a comment.